test/correctness/simd_op_check_powerpc.cpp

```diff
@@ -61,9 +61,11 @@ class SimdOpCheckPowerPC : public SimdOpCheckTest {
             check("vsububs", 16 * w, u8(max(i16(u8_1) - i16(u8_2), 0)));
             check("vsubuhs", 8 * w, u16(max(i32(u16_1) - i32(u16_2), 0)));
             check("vsubuws", 4 * w, u32(max(i64(u32_1) - i64(u32_2), 0)));
-            check("vsububs", 16 * w, absd(i8_1, i8_2));
-            check("vsubuhs", 16 * w, absd(i16_1, i16_2));
-            check("vsubuws", 16 * w, absd(i32_1, i32_2));
+            // TODO: not getting generated in recent LLVM builds.
+            // https://github.com/halide/Halide/issues/7208
+            // check("vsububs", 16 * w, absd(i8_1, i8_2));
```

Comment:
https://github.com/halide/Halide/commit/bc5b5575576e480fdcceb3ab7cc46bdc69e7b64c

It looks like the cause was me accidentally putting these ones in the powerpc section instead of the x86 section! I don't think they're even correct, because the or-two-saturating-subtracts trick only works for unsigned.

Reply:
I'll take the liberty of pushing a fix to this branch, moving these tests to the right place.

Reply:
Wait, x86 doesn't have any instructions of those names, does it?

Reply:
It's vpsubusb on x86, which looked close enough to fool me.